packages/common-integration-tests/src/common-int-test-utils.ts

```diff
@@ -70,3 +70,19 @@ export function ItBehavesLikeItValidatesCacheName(
     );
   });
 }
+
+const bytesEncoderForTests = new TextEncoder();
+
+/**
+ * TODO
+ * TODO
+ * TODO explain
```

Comment:
This was the other major fix in this PR.  Something about how TextEncoder is populating the Uint8Array is different between nodejs SDK and web SDK; I suspect that the Uint8Array class can be backed by either an actual array of ints or some more fancy type of Buffer object, and that if you try to compare two Uint8Arrays that are using a different backing implementation, jest isn't smart enough to figure it out and you get the weird "object vs array" diffs that you were seeing, even though the bytes are the same.

You should document that this function ensures we end up with a Uint8Array backed by a simple array of ints, and that that is better for testing since it ensures Jest will be comparing the things we want it to compare.

You should also probably sweep through the shared int tests and get rid of any references to TextEncoder in favor of this function.  (You may need to add a similar helper for `decode` if any of the tests are relying on that.)